Packs/CarbonBlackDefense/Integrations/CarbonBlackLiveResponseCloud/CarbonBlackLiveResponseCloud.py

```diff
@@ -0,0 +1,319 @@
+import demistomock as demisto
+from CommonServerPython import *
+from cbc_sdk import endpoint_standard
+from cbc_sdk import CBCloudAPI
+from cbc_sdk import errors
+import ntpath
+
+''' IMPORTS '''
+
+import requests
+
+# Disable insecure warnings
+requests.packages.urllib3.disable_warnings()  # pylint: disable=no-member
+
+''' CONSTANTS '''
+
+DATE_FORMAT = '%Y-%m-%dT%H:%M:%SZ'  # ISO8601 format with UTC, default in XSOAR
+CBD_LR_PREFIX = 'cbd-lr'
+
+
+# Using Py API
+def put_file_command(credentials: Dict, sensor_id: int, destination_path: str, file_id: str):
+    api = CBCloudAPI(**credentials)
+    session = api.select(endpoint_standard.Device, sensor_id).lr_session()
+    file_path = demisto.getFilePath(file_id).get('path')
+    session.put_file(open(file_path, 'rb'), destination_path)
+    return f'File: {file_id} is now exist in the remote destination {destination_path}'
+
+
+# Using Py API
+def get_file_command(credentials: Dict, sensor_id: int, source_path: str, timeout: int = None,
+                     delay: float = None):
```

Comment:
```suggestion
                     delay: Union[float, str] = None):
```

Since you are "flatting" the args dictionary, and when you use the CLI in the product the arguments are passed as strings then you get either string or a float.
Change it in all relevant places, and where needed - verify that you get the right type of argument.